python/examples/notebooks/README.md

```diff
@@ -1,36 +1,72 @@
-# BeeAI Python Notebook Examples
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="/docs/assets/Bee_logo_white.svg">
+    <source media="(prefers-color-scheme: light)" srcset="/docs/assets/Bee_logo_black.svg">
+    <img alt="Bee Framework logo" height="90">
+  </picture>
+</p>
 
-These examples organize agentic topics from the BeeAI framework for Python into structured groups, making it easier to learn the framework.
+<h1 align="center">BeeAI Framework Notebooks</h1>
 
-## Pre-requisites
+This series of notebooks will guide you through the features of the BeeAI framework. You'll learn how to create, manage, and optimize AI agents with the help of practical examples and explanations.
 
-We do not assume any particular method/software for executing these notebooks. The minimum requirements are contained within the `requirements.txt` files. These can be installed after activating your Python virtual environment, as follows:
+## 🛠 Setup
+
+First clone the beeai-framework repo. 
 
 ```shell
-pip install -r examples/requirements.txt
-pip install -r examples/notebooks/requirements.txt
+git clone git@github.com:i-am-bee/beeai-framework.git
```

Comment:
Guys without GitHub accounts (local keys) may not use it. Consider switching to HTTP.